app/design/adminhtml/Magento/backend/web/css/styles-old.less

```diff
@@ -989,7 +989,7 @@
             padding-top: 0;
             padding-right: 30px;
             width: 20%;
-            margin-left: 0;
+            margin-left: 30px;
```

Comment:
We need another solution here. This change does not resolve the issue. Please, check my comment in the conversation thread.

Reply:
@rogyar 
Sure i will upload with another solution